packages/backend/src/tokens/__tests__/request.test.ts

```diff
@@ -1132,4 +1210,198 @@ describe('tokens.authenticateRequest(options)', () => {
       expect(refreshSession).toHaveBeenCalled();
     });
   });
+
+  describe('Machine authentication', () => {
+    afterEach(() => {
+      vi.clearAllMocks();
+    });
+
+    // Test each token type with parameterized tests
+    const tokenTypes = ['api_key', 'oauth_token', 'machine_token'] as const;
```

Comment:
I'm mildly confused by the typecasting here

Reply:
Yeah the `as const` is needed here so TS knows these are literal types that match the keys in our mock objects, otherwise it would just see it as `string[]`